R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -576,7 +576,7 @@ test_that("test tableNames and tables", {
   tables <- tables()
   expect_equal(count(tables), 2)
   suppressWarnings(dropTempTable("table1"))
-  dropTempView("table2")
+  expect_true(dropTempView("table2"))
```

Comment:
We have a bunch of other places where we call `dropTempView` - Might be good to add this to those places which we think are testing temp views ?